var/spack/repos/builtin/packages/nmap/package.py

```diff
@@ -46,11 +45,16 @@ def configure_args(self):
         # https://github.com/nmap/nmap/issues/2144
         args.append("--disable-rdma")
 
+        # ndiff and zenmap both require python2, which is deprecated.
+        # The next nmap release will fix this, so, disable these features
+        # completely for now. We will add feature flags for these features again
+        # when they can be supported without a dependency on python2
+        args.append("--disable-ndiff")
+        args.append("--disable-zenmap")
```

Comment:
I like this approach.  Shows these are options and explains they can be re-enabled once the codes support a reasonable (😉 ) Python.